topdown/cache.go

```diff
@@ -19,7 +42,7 @@ type virtualCacheElem struct {
 	undefined bool
 }
 
-func newVirtualCache() *virtualCache {
+func NewVirtualCache() VirtualCache {
```

Comment:
This is being used by the debugger implementation hence we're making this public. Couldn't we use an internal package for this and avoid exposing this?

Reply:
Nvm. I just saw we're doing `func (q *Query) WithVirtualCache(vc VirtualCache) *Query`